### DISCUSSION TITLE
Fare Policy Criteria's and Standardization

### DISCUSSION THREAD

**Discussion by 92shreyansh** [ONDC-Official/mobility-specification#40] [Ideas]
Hi All,

Lets use this thread to discuss and identify possible criteria's for fare policy and its calculation which would help the Network to streamline and make the values machine computable.

**Comment by sashokkumarsaravanan** [comment]
Dear ONDC Community,

We're excited to share that as part of our ongoing efforts to enhance user experience and streamline our services we're implementing a new set of fare matrix for scheduled rides within the ONDC network. 
Our recommendation for the community is to review the matrix and happy to discuss further. 

Scheduled ride rate matrix and applicable attributes: 

Version - 1

Fare breakups
1. Driver fare (Per day/250 km for outstation trips).
2. Toll charge.
3. Hill station charge.
4. Waiting charge.
5. Airport entry charge.
6. Airport parking charge.
7. Night charge.
8. State tax.
9. Other charge

Sample:

Price per km: 22, Additional km charge: 19

Base fare:1000
BuyerCommsion:200
GST:120
Grand total:1320

tollCharge: {
inclusive: false
amount: 0
}

waitingCharge: {
inclusive: false
applicableTime: 30
freeWaitingTime: 30
amount: 0
}

nightCharge: {
inclusive: false
applicableTime: 30
freeWaitingTime: 30
amount: 0
} 
otherCharge: [
inclusive: false
name: ‘’
amount: 0
]

Warm regards,

Ashokkumar Saravanan
On behalf of Taxida Private Limited
ONDC Community Team.

**Comment by sashokkumarsaravanan** [comment]
We need better clarity on the following for scheduled rides:

1. How can we rise the package/Hourly rental booking request from the buyer.
2. How can we rise the round trip booking request from the buyer app.
3. Purpose of messageId in context object 
4. How formulate the bpp_uri in context object
